FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -619,4 +624,60 @@ - (FIRConfigUpdateListenerRegistration *)addOnConfigUpdateListener:
   return [self->_configRealtime addConfigUpdateListener:listener];
 }
 
+#pragma mark - Rollout
+
+- (void)addRemoteConfigInteropSubscriber:(id<FIRRolloutsStateSubscriber>)subscriber {
+  [[NSNotificationCenter defaultCenter]
+      addObserverForName:RolloutsStateDidChangeNotificationName
+                  object:self
+                   queue:nil
+              usingBlock:^(NSNotification *_Nonnull notification) {
+                FIRRolloutsState *rolloutsState =
+                    notification.userInfo[RolloutsStateDidChangeNotificationName];
+                [subscriber rolloutsStateDidChange:rolloutsState];
+              }];
+  // Pass active rollout metadata stored in persistence if exists while app launched.
+  NSArray<NSDictionary *> *activeRolloutMetadata = self->_configContent.activateRolloutMetadata;
+  dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+    [self notifyRolloutsStateChange:activeRolloutMetadata
+                      versionNumber:self->_settings.lastActiveTemplateVersion];
+  });
+}
+
+- (void)notifyRolloutsStateChange:(NSArray<NSDictionary *> *)rolloutMetadata
+                    versionNumber:(NSString *)versionNumber {
```

Comment:
I think you could omit the version number check, too. I don't know a case when it'd be 0 practically, but I think we'd want to pass it along regardless

Reply:
When the app first installed without any fetch, the version number will be 0. The first notification send to Crashlytics will be an empty rollout metadata list, which I think is also acceptable. If an activation happen later, a new rollout metadata will append to the first empty one, which would be uploaded if any crash happen.

Reply:
Updated to check activeConfig instead of active template version number.